### PR TITLE
Spk 37704 missing areacodes

### PR DESCRIPTION
**Pull request by juanbandajr17**

### CODE REVIEW

File: spec/lib/areacodes_spec.rb [hunk 1]
```diff
@@ -69,6 +70,33 @@
...
+          ['New Hampshire', 1, 33, [11, 13, 17, 5, 15, 1, 9, 19, 7, 3]],
+          ['Connecticut', 4, 9, [1, 9, 5, 3, 7, 11, 15, 13]],
+          ['New Jersey', 9, 34, [1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39, 41]],
+          ['Rhode Island', 1, 44, [1, 3, 5, 7, 9]],
+          ['Vermont', 1, 50, [1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27]],
+        ]
+      end
+
+      it 'has area codes for state and counties' do
+        state_and_counties.each do |name, mincount, state, counties|
```
Comment (rdadoune) [inline spec/lib/areacodes_spec.rb]: It's not exactly clear to me what purpose `mincount` serves, what is it for?

Comment (juanbandajr17) [reply spec/lib/areacodes_spec.rb]: The ticket https://spokeo.atlassian.net/browse/SPK-37704 lists the number of area-codes per state. The test counts all the area-codes from all the counties within the state and makes sure it doesn't fall below the number displayed on the ticket.

I'll make it more robust to collect the area-codes, then apply a unique filter, and then count.

Comment (juanbandajr17) [reply spec/lib/areacodes_spec.rb]: But this test can start failing if area-codes start dropping out of nanpa ... this test might be a bit overkill I guess.

Comment (rdadoune) [reply spec/lib/areacodes_spec.rb]: Ohh okay I see

**Pull request merged**